.github/workflows/html-editor-checks.yml

```diff
@@ -0,0 +1,104 @@
+name: Html Editor
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  set-up-branches:
+    name: devextreme branches setup
+    runs-on: ubuntu-latest
+    outputs:
+      branches: ${{ steps.set-matrix.outputs.branches }}
+    steps:
+      - name: Checkout Quill repository
+        uses: actions/checkout@v3
+      - id: set-matrix
+        run: |
+          BRANCHES=$(echo $(cat .github/workflows/devextreme-branches.json) | sed 's/ //g')
+          echo $BRANCHES
+          echo "branches=$BRANCHES" >> $GITHUB_OUTPUT
```

Comment:
Is it necessary to put a file with branches and then read it here? Why don't you just inline a list in this workflow?

Reply:
Fixed